frontend/src/Task.js

```diff
@@ -1,22 +1,135 @@
-import { FaTrashAlt } from "react-icons/fa";
+import { useAtom } from "jotai";
+import { useEffect, useRef, useState } from "react";
+import { createPortal } from "react-dom";
+import { FaCaretDown } from "react-icons/fa";
+import { useUpdateTask } from "./api/api.hooks";
+import { openMenuAtom } from "./atoms";
 
-import { Button } from "./Button";
+const badgeStyle = {
```

Comment:
According to the React docs, there is no consensus about how styles should be defined, but at the same time it states that CSS classes are generally better for performance than inline styles and probably also CSS-in-JS, which is not a part of React, but provided by third-party libraries.
With that in mind, can we move all these styles and classes to css files?
I think a good use of CSS-in-JS is when we need to use math depending on the React state or JS specifics, which does not seem to be the case here so I think it is safe to proceed in this way. 
What do you think?